docs/decisions/0007-sdn-sancation-state-on-orders-in-commercetools.rst

```diff
@@ -0,0 +1,41 @@
+7. SDN Sancationed State on Orders in Commercetools for R0.1
+############################################################
+
+Status
+******
+
+**Provisional** *2023-11-29* -- R0.1
+
+Context
+*******
+
+Due to the speed required for R0.1, we needed a way to mark orders as blocked by an SDN match.
+
+This will be implemented by a single `custom State`_ within the Order. This is sometimes referred to as a [Workflow] Transition State.
```

Comment:
Why not use [PaymentStatus](https://docs.commercetools.com/api/projects/payments#ctp:api:type:PaymentStatus)?

Reply:
An Order's workflow state is more searchable in the Merchant Center than a Payment's workflow state.